versioned_hdf5/meson.build

```diff
@@ -26,6 +26,29 @@ py.extension_module(
     override_options : ['cython_language=cpp'],
 )
 
+cython_args = [
+    # '-a',  # Generate HTML report
+    '-X', 'infer_types=True',
+    # Comment these out if you experience segfaults to get a clear exception instead
+    '-X', 'initializedcheck=False',
+    '-X', 'boundscheck=False',
+    # Note: this is possible because we went through all view accesses in the module
+    # and made sure they always use non-negative indices.
+    '-X', 'wraparound=False',
+    # Note: this is possible only because we went through every single // and % in
+    # the module and made sure the sign of the two operands is always the same.
+    # Comment this out if behaviour changes when not compiled.
+    '-X', 'cdivision=True',
+]
+
+# To generate HTML compilation reports:
+# (First double check that flags below match those above)
+#
+# cythonize -3 -a -X infer_types=True -X initializedcheck=False -X boundscheck=False -X wraparound=False -X cdivision=True versioned_hdf5/{hyperspace,subchunk_map}.py
+#
+# Or uncomment the -a flag above and then run:
+# meson build && pushd build && meson compile && popd && firefox $(find build/ -name "*.html")
+
```

Comment:
All of the above is a copy-paste from #371 - replicated here to allow for parallel review